src/contractor/contractor.cpp

```diff
@@ -95,10 +98,11 @@ EdgeWeight getNewWeight(IterType new_speed_iter, const double segment_length, Ed
     {
```

Comment:
regarding this change. I am a bit sceptical for the verbosity we might see. If a traffic jam happens, we can easily see a factor of 2.5. But if that jam dissolves, we would get a strong increase again. Can we maybe add some factor here that considers our fallback on lua to check if we increase strongly above a reasonable threshold?